src/control/common/proto/mgmt/addons.go

```diff
@@ -87,22 +87,56 @@ func (r *DeleteACLReq) SetSvcRanks(rl []uint32) {
 }
 
 func Debug(msg proto.Message) string {
-	switch t := msg.(type) {
+	var bld strings.Builder
+	switch m := msg.(type) {
 	case *SystemQueryResp:
 		stateRanks := make(map[string]*system.RankSet)
-		for _, m := range t.Members {
+		for _, m := range m.Members {
 			if _, found := stateRanks[m.State]; !found {
 				stateRanks[m.State] = &system.RankSet{}
 			}
 			stateRanks[m.State].Add(system.Rank(m.Rank))
 		}
-		var bld strings.Builder
-		fmt.Fprintf(&bld, "%T ", t)
+		fmt.Fprintf(&bld, "%T ", m)
 		for state, set := range stateRanks {
 			fmt.Fprintf(&bld, "%s: %s ", state, set.String())
 		}
-		return bld.String()
+	case *PoolCreateReq:
+		fmt.Fprintf(&bld, "%T uuid:%s u:%s g:%s ", m, m.Uuid, m.User, m.Usergroup)
+		if len(m.Properties) > 0 {
+			fmt.Fprintf(&bld, "p:%+v ", m.Properties)
+		}
```

Comment:
nit - would be better if these labels were spelled out slightly more for future developers, even if it's just debug output.